app/src/main/java/uk/ac/ebi/atlas/search/JsonGeneSearchController.java

```diff
@@ -236,7 +238,7 @@ public Boolean isMarkerGene(@RequestParam MultiValueMap<String, String> requestP
                                 .map(Map.Entry::getKey)
                                 .collect(toImmutableSet()));
 
-        return markerGeneFacets != null && markerGeneFacets.size() > 0;
+        return markerGeneFacets != null && !markerGeneFacets.isEmpty();
```

Comment:
We can use one statement `StringUtils.isBlank()`-> this would check white spaces and null.